search-ui/src/App.vue

```diff
@@ -16,14 +16,14 @@
 
     <loading-screen v-if="appLoading" :is-loading="appLoading" />
     <sbc-header v-if="auth.tokenInitialized" class="sbc-header" :in-auth="false" :show-login-menu="false" />
-    <bcrs-breadcrumb :breadcrumbs="breadcrumbs" v-if="breadcrumbs.length > 0" />
     <sbc-system-banner
       v-if="systemMessage != null"
       class="justify-center"
       :setShow="systemMessage != null"
-      :setType="systemMessageType"
       :setMessage="systemMessage"
+      :setMessageType="systemMessageType"
```

Comment:
this prop should be :setType=...

you can just remove this line (it will default to 'warning') or replace it with `setType="warning"`

Reply:
Ok, done.